assets/mosaic.yaml

```diff
@@ -21,3 +21,24 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: ''
+- type: model
+  name: CommonCanvas
+  organization: Cornell University, Mosaic
+  description: CommonCanvas is a text-to-image model trained solely on Creative Commons licensed images.
+  created_date: 2023-10-25
+  url: https://arxiv.org/pdf/2310.16825.pdf
+  model_card: none
+  modality: text; image
+  analysis: Compared to Stable Diffusion 2, a SOTA text-to-image model.
+  size: unknown
+  dependencies: [Creative Commons]
```

Comment:
CreativeCommons is a license type (for data) not a source. The dataset described in https://arxiv.org/pdf/2310.16825.pdf is called CommonCatalog (which should have its own node in EG).